### DISCUSSION TITLE
Bing suggestions URL?

### DISCUSSION THREAD

**Discussion by choc1024** [Eloston/ungoogled-chromium#2828] [Q&A] [answered]
I want to add the bing search engine (accidentally deleted) what is the bing search suggestions URL?

**Comment by touba-denrael** [comment, marked as answer]
under debian bookworm:

`https://www.bing.com/search?q=%s`

`https://www.bing.com/osjson.aspx?query=%s`

hope this helps with your question

**Reply by PF4Public** [reply]
Shouldn't you omit `PC=` part?

**Reply by touba-denrael** [reply]
just copied the strings from the bing settings fields [using startpage myself, sorry]

**Reply by PF4Public** [reply]
I just suspect that `PC=` might be used for tracking :)

**Reply by touba-denrael** [reply]
should the copied strings be edited as you suggest [omitting those pc=xxxx parts]?

**Reply by PF4Public** [reply]
I would keep only `q=%s` to be sure.

**Reply by touba-denrael** [reply]
ashamed to bother you again, does the present version seem to be the correct one?

thanks for all your kind help, @PF4Public.

**Reply by PF4Public** [reply]
Yep, looks good enough:
![image](https://github.com/ungoogled-software/ungoogled-chromium/assets/10319700/9ee1279b-e8e3-42cb-87fd-1a9fdd757dc1)

**Reply by touba-denrael** [reply]
at long last [through couple of iterations though], thanks again, @PF4Public.

**Reply by choc1024** [reply]
thanks, solved the problem. Sorry for the late reply. Closing the discussion.

**Reply by touba-denrael** [reply]
only with, through, the expert help of @PF4Public, without that, my answer would have been the worst, simplest, copy paste operation.

thanks, @EinKaffeeBitte, also many, again, to @PF4Public.
